src/game_io/game_loader.cc

```diff
@@ -91,11 +91,10 @@ int32_t GameLoader::load_game(bool const multiplayer) {
 	FileSystem* map_fs = game_.map().filesystem();
 	if (map_fs->file_exists("scripting/tribes")) {
 		log_info("Game: Reading Scenario Tribes ... ");
-		game_.mutable_tribes()->register_scenario_tribes(map_fs);
+		game_.mutable_descriptions()->register_scenario_tribes(map_fs);
+		log_info("Game: Reading Scenario Tribes took %ums\n", timer.ms_since_last_query());
```

Comment:
no need for the second log line

Reply:
Why don't you want it? This is what it looks like in context:

```
[00:01:04.992 real] INFO: Game: Reading Scenario Tribes ... 
[00:01:05.041 real] INFO: Registering the descriptions took 49ms
[00:01:05.055 real] INFO: Game: Reading Scenario Tribes took 65ms
[00:01:05.055 real] INFO: Game: Reading Player Info ...
[00:01:05.528 real] INFO: ┏━ Loading empire
[00:01:05.528 real] INFO: ┃    Ships
[00:01:05.531 real] INFO: ┃    Immovables
[00:01:05.538 real] INFO: ┃    Wares
[00:01:05.557 real] INFO: ┃    Workers
[00:01:05.728 real] INFO: ┃    Buildings
[00:01:05.776 real] INFO: ┃    Frontiers, flags and roads
[00:01:05.777 real] INFO: ┃    Finalizing
[00:01:05.778 real] INFO: ┗━ took 250ms
[00:01:05.797 real] INFO: ┏━ Loading barbarians
[00:01:05.797 real] INFO: ┃    Ships
[00:01:05.800 real] INFO: ┃    Immovables
[00:01:05.803 real] INFO: ┃    Wares
[00:01:05.820 real] INFO: ┃    Workers
[00:01:05.866 real] INFO: ┃    Buildings
[00:01:05.909 real] INFO: ┃    Frontiers, flags and roads
[00:01:05.910 real] INFO: ┃    Finalizing
[00:01:05.910 real] INFO: ┗━ took 113ms
[00:01:05.911 real] INFO: Game: Calling read_complete()
[00:01:05.917 real] INFO: Reading Elemental Data ... 

```

Reply:
```
[00:01:04.992 real] INFO: Game: Reading Scenario Tribes ... 
[00:01:05.041 real] INFO: Registering the descriptions took 49ms
[00:01:05.055 real] INFO: Game: Reading Scenario Tribes took 65ms
```
No need to print the first line again. We got rid of such duplications everywhere else when switching to the new log output formatting.

Reply:
I see where you're coming from. Shall we remove the following line too?

`log_info("Game: read_complete took: %ums\n", timer.ms_since_last_query());`

Reply:
Done.